cdap-data-fabric/src/test/java/co/cask/cdap/data/hbase/HBaseTestBase.java

```diff
@@ -85,7 +86,18 @@ private void startHBase() throws Exception {
     getConfiguration().setInt("hbase.regionserver.port", Networks.getRandomPort());
     getConfiguration().setInt("hbase.regionserver.info.port", Networks.getRandomPort());
 
-    doStartHBase();
+    for (int startAttempt = 0; startAttempt < MAX_START_ATTEMPTS; startAttempt++) {
+      try {
+        doStartHBase();
+        LOG.info("Successfully started HBase");
+        break;
+      } catch (IOException e) {
+        LOG.error("Failed to start HBase", e);

```

Comment:
should probably say "retrying" and then before throwing error, we can log  "couldn't start after x attempts"